src/mgmt/srv_target.c

```diff
@@ -295,6 +411,10 @@ tgt_vos_create_one(void *varg)
 
 	if (path)
 		D_FREE(path);
+	/* kccain experiment: kill server here - will server restart
```

Comment:
temporary code to activate to trigger scenario in which SPDK blobs need to be cleaned up on next server start